src/atlas/trans/Trans.cc

```diff
@@ -125,6 +125,28 @@ void Trans::invtrans_vordiv2wind( const Field& spvor, const Field& spdiv, Field&
     get()->invtrans_vordiv2wind( spvor, spdiv, gpwind, options( config ) );
 }
 
+
+void Trans::invtrans_adj( const Field& gpfield, Field& spfield, const eckit::Configuration& config ) const {
+    get()->invtrans_adj( gpfield, spfield, options( config ) );
+}
+
+void Trans::invtrans_adj( const FieldSet& gpfields, FieldSet& spfields, const eckit::Configuration& config ) const {
+    get()->invtrans_adj( gpfields, spfields, options( config ) );
+}
+
+void Trans::invtrans_adj_grad( const Field& gradfield, Field& spfield, const eckit::Configuration& config ) const {
+    get()->invtrans_adj_grad( gradfield, spfield, options( config ) );
+}
+
+void Trans::invtrans_adj_grad( const FieldSet& gradfields, FieldSet& spfields, const eckit::Configuration& config ) const {
+    get()->invtrans_adj_grad( gradfields, spfields, options( config ) );
+}
+
+void Trans::invtrans_adj_vordiv2wind( const Field& gpwind, Field& spvor, Field& spdiv,
+                                     const eckit::Configuration& config ) const {
+    get()->invtrans_adj_vordiv2wind( gpwind, spvor, spdiv, options( config ) );
+}
+
```

Comment:
Should we not rather have the "_adj" as a function suffix rather than in the middle?

Reply:
Done!

Reply:
Done